Makefile

```diff
@@ -168,7 +168,6 @@ TGTS=	all all-man buildenv buildenvvars buildkernel buildsysroot buildworld \
 	build32 distribute32 install32 \
 	build64 distribute64 install64 \
 	build64c distribute64c \
-	libcheribuildenv libcheribuildenvvars \
```

Comment:
We should also update cheribuild to keep --subdir working: https://github.com/CTSRD-CHERI/cheribuild/blob/985d7f3ed124e55e776b3d018917d7101548c6f8/pycheribuild/projects/cross/cheribsd.py#L1344

Reply:
Yes, https://github.com/CTSRD-CHERI/cheribuild/pull/229 probably needs to merged before this.  I could probably go ahead and do that now.